bodhi/server/renderers.py

```diff
@@ -105,8 +105,11 @@ def link_dict(obj):
             for name, getter in getters[key].items():
                 # Because we have to use methods to fill feed entry attributes,
                 # it's done by getting methods by name and calling them
-                # on the same line
-                getattr(feed_item, name)(getter(value))
+                # on the same line, alongwith setting generalized comment title.
+                if key == "comments" and name == "title":
+                    getattr(feed_item, name)("{} comment #{}".format(value.update.alias, value.id))
+                else:
+                    getattr(feed_item, name)(getter(value))
```

Comment:
I think it would actually be best to always use the ```"{} comment #{}"``` template, even when the comment does have text, for consistency.

Rather than putting this special code here to specialcase the comment title, why not make a custom getter? Or, you could also add a ```property``` to the ```Comment``` class and change ```getters['comments']['title']``` above to use that new property instead of the ```text``` like it does now.